scripts/redux/constants.js

```diff
@@ -7,3 +7,7 @@ const SET_HERO = 'SET_HERO';
 const FETCH_TICKETS = 'FETCH_TICKETS';
 // partners
 const FETCH_PARTNERS = 'FETCH_PARTNERS';
+// featured videos
+const FETCH_VIDEOS = 'FETCH_VIDEOS';
+// video dialog
+const TOGGLE_DIALOG = 'TOGGLE_DIALOG';
```

Comment:
Can we use this constant to invoke other dialogs, like speaker's, session's, etc. ?